lib/LaTeXML/Util/Unicode.pm

```diff
@@ -24,6 +23,73 @@ sub UTF {
   my ($code) = @_;
   return pack('U', $code); }
 
+my $NBSP = UTF(0xA0);
+sub NBSP { return $NBSP; }
```

Comment:
Is this attempting to guard the lexical `$NBSP` from some sort of external redefinition?

Surprised it wasn't introduced as a package-level `our $NSBP;` declaration. That can be added to `@EXPORT` too, if desired.

Reply:
Thanks, switched over to cleaner `\N{NBSP}` approach (along with `use charnames ':full';` for older perls)